src/main/java/no/unit/nva/database/UserDb.java

```diff
@@ -32,9 +32,12 @@
     public static final String INVALID_PRIMARY_HASH_KEY = "PrimaryHashKey of user should start with \"USER\"";
 
     private String primaryHashKey;
+
     private String username;
     private String institution;
     private List<RoleDb> roles;
+    private String givenName;
```

Comment:
It strikes me that other cultures don't have a given name / family name split, this may or may not be modelled in the data ssource

Reply:
If the familyName is empty is it a problem? If not then if someone does not have a family name then we just fill in the firstName